Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.kt

```diff
@@ -206,7 +206,10 @@ class InputOverlay(context: Context?, attrs: AttributeSet?) : SurfaceView(contex
                         pressed = true
                     }
                 }
-
+            }
+            when (event.action and MotionEvent.ACTION_MASK) {
+                MotionEvent.ACTION_DOWN,
+                MotionEvent.ACTION_POINTER_DOWN,
```

Comment:
Why don't you just move this code block into the section above and just add `MotionEvent.ACTION_MOVE` as one of the conditions to trigger it?

Reply:
We don't want to run `dpad.trackId = event.getPointerId(pointerIndex)` on ACTION_MOVE, since that would mean that sliding from a button over to the d-pad would trigger the d-pad.